crates/aptos/src/account/create_resource_account.rs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::common::types::{CliCommand, CliTypedResult, TransactionOptions};
+use aptos_rest_client::{
+    aptos_api_types::{WriteResource, WriteSetChange},
+    Transaction,
+};
+use aptos_transaction_builder::aptos_stdlib;
+use aptos_types::account_address::AccountAddress;
+use async_trait::async_trait;
+use clap::Parser;
+use serde::Serialize;
+
+/// Command to create a resource account
+///
+#[derive(Debug, Parser)]
+pub struct CreateResourceAccount {
+    #[clap(flatten)]
+    pub(crate) txn_options: TransactionOptions,
+
+    /// Resource account seed.
+    #[clap(long)]
+    pub(crate) seed: String,
```

Comment:
is this intended to be an arbitrary string or a hex string that is the actual seed? It might be good to define how this function works, so that someone could compute the potential account... it might also be good to make a dry-run function here, so that a user can extract the account number without actually creating it.

Reply:
I think additionally a comment about what this seed does would help a user.

```
/// Resource account seed
///
/// Seed used in generation of the AccountId of the resource account
```

Reply:
Documentation is added with further clarification on the string usage (it will first get serialised into a `vec<u8>` via `BCS` then uses for generating the `ResourceAccount`'s `AccountId`.). For the dry run feature, I feel it will be nice to do it in a separate PR if possible.